.circleci/config.yml

```diff
@@ -402,16 +372,8 @@ workflows:
             branches:
               only:
                 - main
-      - test_staging:
-          requires:
-            - deploy_staging
-          filters:
-            branches:
-              only: main
       - deploy_production_approval:
           type: approval
-          requires:
-            - test_staging
```

Comment:
We still need a requires step or it'll try to get the deploy_production_approval straight away since it doesn't think anything comes first. Order is not respected in CircleCI, only requirement dependencies.

So it should be changed to:

```yaml
          requires:
            - deploy_staging
```